### DISCUSSION TITLE
How to stream videos like in the demo?

### DISCUSSION THREAD

**Discussion by pini85** [webtorrent/webtorrent#2387] [Q&A]
Hi guys. I've been trying to stream a video from a torrent to no avail.
I'm trying in several ways.
I have for example this torrent file:
```
 {
   time: "Dec. 22nd '21",
   seeds: 8688,
    peers: 957,
     size: '2.8 GB',
    desc: 'XXXXXX'
  }
```
 
1) Im getting the torrent magnet, placing it as the torrentId, it is not loading the video. Example of a magnet is this:
'XXXXXXXXX'

Also tried parseTorrent and torrentStream npm packages. Can anybody shed some light on what I am doing wrong? Would highly appreciate it! Thanks!

**Comment by DiegoRBaquero** [comment]
Please don’t use Webtorrent for copyright infringement DiegoOn 11/10/2022, at 5:14 AM, Pinchas Hodadad ***@***.***> wrote:﻿
Hi guys. I've been trying to stream a video from a torrent to no avail.
I'm trying in several ways.
I have for example this torrent file:
 {
   time: "Dec. 22nd '21",
   seeds: 8688,
    peers: 957,
     size: '2.8 GB',
    desc: 'https://www.1337x.to/torrent/5093050/The-Matrix-4-Resurrections-2021-1080p-WEBRip-x264/',
   provider: '1337x'
  }


Im getting the torrent magnet, placing it as the torrentId, it is not loading the video. Example of a magnet is this:
'magnet:?xt=urn:btih:CD30EFBEB096CD88DAFA1E761300A6566EB66E6F&dn=The.Matrix.4.Resurrections.2021.1080p.WEBRip.x264&tr=http%3A%2F%2Ftracker.trackerfix.com%3A80%2Fannounce&tr=udp%3A%2F%2F9.rarbg.me%3A2780%2Fannounce&tr=udp%3A%2F%2F9.rarbg.to%3A2820%2Fannounce&tr=udp%3A%2F%2Ftracker.fatkhoala.org%3A13790%2Fannounce&tr=udp%3A%2F%2Ftracker.tallpenguin.org%3A15710%2Fannounce&tr=udp%3A%2F%2Ftracker.opentrackr.org%3A1337%2Fannounce&tr=http%3A%2F%2Ftracker.openbittorrent.com%3A80%2Fannounce&tr=udp%3A%2F%2Fopentracker.i2p.rocks%3A6969%2Fannounce&tr=udp%3A%2F%2Ftracker.internetwarriors.net%3A1337%2Fannounce&tr=udp%3A%2F%2Ftracker.leechers-paradise.org%3A6969%2Fannounce&tr=udp%3A%2F%2Fcoppersurfer.tk%3A6969%2Fannounce&tr=udp%3A%2F%2Ftracker.zer0day.to%3A1337%2Fannounce'

Also tried parseTorrent and torrentStream npm packages. Can anybody shed some light on what I am doing wrong? Would highly appreciate it! Thanks!
